aodndata/soop/soop_trv.py

```diff
@@ -28,29 +32,16 @@ def preprocess(self):
 
     def get_main_soop_trv_var(self, filepath):
         netcdf_file_obj = Dataset(filepath, mode='r')
-        variables = netcdf_file_obj.variables.keys()
+        nc_variables = netcdf_file_obj.variables.keys()
         netcdf_file_obj.close()
 
-        if 'CPHL' in variables:
-            return 'CPHL'
-        elif 'TEMP' in variables:
-            return 'TEMP'
-        elif 'PSAL' in variables:
-            return 'PSAL'
-        elif 'TURB' in variables:
-            return 'TURB'
+        main_var = [var for var in nc_variables if var in self.var_dict.keys()]
```

Comment:
Same here

Reply:
also here nc_variables is an array of variables. It's trying to compare 2 arrays of string. so not sure how I could use the get